molecule_vagrant/modules/vagrant.py

```diff
@@ -517,6 +535,8 @@ def main():
             platform_box=dict(type="str", required=False),
             platform_box_version=dict(type="str"),
             platform_box_url=dict(type="str"),
+            platform_box_download_checksum=dict(type="str"),
+            platform_box_download_checksum_type=dict(type="str"),
             provider_name=dict(type="str", default="virtualbox"),
```

Comment:
Maybe I'm too picky, but I would have added a requirement on platform_box_download_checksum_type if platform_box_download_checksum is specified, as it's how vagrant is working

Reply:
Good point.